geonode_mapstore_client/client/version.txt

```diff
@@ -1 +1 @@
-geonode-mapstore-client-v2.0.9-25eb58770a416d5861c1bd95637054f63e838f13
\ No newline at end of file
+geonode-mapstore-client-v2.0.9-25eb58770a416d5861c1bd95637054f63e838f13
```

Comment:
I don't understand why is highlighted as changed, probably CRLF/LF

Reply:
Just checked, there are no differences with the master one... probably is CRLF